services/QuillLMS/spec/services/analytics/segment_analytics_spec.rb

```diff
@@ -70,19 +70,28 @@
 
   context 'tracking activity completion' do
     let(:teacher) { create(:teacher) }
+    let(:topic) { create(:topic, level: 1) }
+    let(:second_topic) { create(:topic, level: 1) }
     let(:activity) { create(:diagnostic_activity) }
     let(:student) { create(:student) }
     let(:activity_session) { create(:activity_session) }
 
     it 'sends an event with information about the activity' do
+      create(:activity_topic, topic: topic, activity: activity)
+      create(:activity_topic, topic: second_topic, activity: activity)
       analytics.track_activity_completion(teacher, student.id, activity, activity_session)
       expect(identify_calls.size).to eq(0)
-      expect(track_calls.size).to eq(1)
+      expect(track_calls.size).to eq(2)
       expect(track_calls[0][:event]).to eq(SegmentIo::BackgroundEvents::ACTIVITY_COMPLETION)
       expect(track_calls[0][:user_id]).to eq(teacher.id)
       expect(track_calls[0][:properties][:activity_name]).to eq(activity.name)
       expect(track_calls[0][:properties][:tool_name]).to eq('Diagnostic')
       expect(track_calls[0][:properties][:student_id]).to eq(student.id)
+      expect(track_calls[0][:properties][:topic_level_one]).to eq(topic.name)
+      expect(track_calls[0][:properties][:topic_level_two]).to eq(Topic.find(topic.parent_id).name)
+      expect(track_calls[0][:properties][:topic_level_three]).to eq(Topic.find(Topic.find(topic.parent_id).parent_id).name)
+      expect(track_calls[1][:event]).to eq(SegmentIo::BackgroundEvents::ACTIVITY_COMPLETION)
+      expect(track_calls[1][:properties][:topic_level_one]).to eq(second_topic.name)
```

Comment:
For consistency, what about 
`expect(track_calls[1][:properties][:topic_level_two])` and 
`expect(track_calls[1][:properties][:topic_level_three])`